### PR TITLE
Create What's New for 0.32

### PR DESCRIPTION
**Pull request by pshipton**
Content copied from Issue https://github.com/eclipse-openj9/openj9-website/issues/308

Also includes https://github.com/eclipse-openj9/openj9-website/pull/309

Closes https://github.com/eclipse-openj9/openj9-website/issues/308

### CODE REVIEW

**Comment by pshipton** [comment]
Jenkins website stage

**Comment by pshipton** [comment]
Jenkins website stage

**Comment by pshipton** [comment]
Jenkins website stage

**Comment by keithc-ca** [comment]
@AdamBrousseau can you correct the typo in your commit message, please ('id' -> 'is')?

File: news-page-content/latestRelease.md [hunk 1]
```diff
@@ -22,31 +22,30 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
-- There is a known problem creating system core files on macOS 12.
-
-- To read more about these and other changes, see the [OpenJ9 user documentation](docs/openj9_releases/).
-
-#### Other performance highlights ####
-
-- 14% better write-barrier sequence on Power
-
-- 15-30% better String.indexOf() performance on Power10
+- External decimal conversions acceleration with the Vector Packed Decimal Facility in the data access accelerator library
```
Comment (keithc-ca) [inline news-page-content/latestRelease.md:50]: typo? 'conversions' -> 'conversion'

**Comment by pshipton** [comment]
Jenkins website stage

**Comment by pshipton** [comment]
Jenkins website stage

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**